index.bs

```diff
@@ -372,6 +395,7 @@ The <dfn iterator for=FontIterator>asynchronous iterator initialization steps</d
 1. For each local font |font| on the system, run these steps:
     1. Let |representation| be a [=/font representation=] for |font|.
     1. [=queue/Enqueue=] |representation| to [=/this=]'s {{FontIterator/[[FontList]]}}.
+1. Sort {{FontIterator/[[FontList]]}} in [=list/sort in ascending order|ascending order=] by using {{FontMetadata/postscriptName}} as the sort key.
```

Comment:
nit: https://infra.spec.whatwg.org/#list-sort-in-ascending-order is not in-place, it returns a new list.

Reply:
I see what you're saying RE: being more like code. Thanks!